js/scripts.js

```diff
@@ -0,0 +1,20 @@
+$(function() {
+    var url = 'https://api.icndb.com/jokes/random';
+   
+    function getJoke() {
+        var xhr = new XMLHttpRequest();
+        xhr.open('GET', url);
+        xhr.addEventListener('load', function() {
+            var response = JSON.parse(xhr.response);
+            paragraph.innerHTML = response.value.joke;
+        });
+        xhr.send();
+    }
+
+    var button = document.getElementById('get-joke');
+    button.addEventListener('click', function() {
+        getJoke();
+    });
+    var paragraph = document.getElementById('joke');
+    button.click();
```

Comment:
what this line should do?

Reply:
show a joke after loading page without click the button